js/build.js

```diff
@@ -1569,10 +1588,10 @@ Fliplet.Widget.instance('chat', function (data) {
         $messagesHolder.html('');
         viewConversation(newConversation);
       });
-    }).catch(function(err) {
+    }).catch(function(error) {
       $('.contacts-done-holder').removeClass('creating');
-      $wrapper.find('h4').text(err.message || err.description || err);
-      if (err.indexOf('Participant not found') !== -1) {
+      $wrapper.find('h4').text(Fliplet.parseError(error) || 'Error creating conversation');
```

Comment:
@hcarneiro I suggest following a similar flow as https://developers.fliplet.com/API/fliplet-ui-toast.html#use-a-toast-notification-to-show-error-messages where you show users the 'Error creating conversation' message and then an optional link to see the technical details (as provided by Fliplet.parseError(error)).